apps/demo-game/src/pages/admin/reports/[id].tsx

```
@@ -0,0 +1,458 @@
+                              <div key={segmentIx} className="min-w-12 ">
+                                P{periodIndex + 1} S{segmentIx + 1}
+                              </div>
+                            )
+                          }
+                        )}
+                      </TableCell>
+                      {Object.values(dataPerPlayer).map((d) => {
+                        const decisions = d.decisions
+
+                        return (
+                          <TableCell className="align-top font-medium">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)